lib/Components/Cart/cart.css

```diff
@@ -0,0 +1,211 @@
+@import "/lib/Components/core.css";
+@import "/lib/Components/utilities.css";
+@import "/lib/Components/Header/header.css";
+@import "/lib/Components/Footer/footer.css";
+
+:root {
+    --COLOR-RED-1: 255, 0, 0;
```

Comment:
Naming - We should not be using variable names with a specific value like "red", "green" etc as they are not flexible to use later on